codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/HttpProtocolUnitTestRequestGenerator.java

```diff
@@ -282,7 +274,8 @@ protected void generateTestServer(
             String name,
             Consumer<GoWriter> handler
     ) {
-        super.generateTestServer(writer, name, handler);
+        // We aren't using a test server, but we do need a URL to set.
```

Comment:
nit: actually this comment doesnt do a lot of good when its generated. because the generated code wont have this comment. if we can, we should make the URL value even more obvious that its a placeholder (while still being parse-able by whatever needs it). for example, would `http://thisisa.placeholder:4242` work?